parley/src/layout/layout.rs

```diff
@@ -35,6 +35,11 @@ impl<B: Brush> Layout<B> {
         &self.data.styles
     }
 
+    /// Returns available width of the layout.
```

Comment:
Could you please expand the documentation of `available_width`, `width`, and `full_width` so that it's clear how each differ?